desktop/libs/notebook/src/notebook/connectors/sql_alchemy_tests.py

```diff
@@ -219,6 +219,23 @@ def test_check_status(self):
       assert_equal(response['status'], 'available')
 
 
+  def test_explain(self):
```

Comment:
Written a raw code for the test, not getting how to write properly please guide me.